modules/apigee_edge_actions/apigee_edge_actions.tokens.inc

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * @file
+ * Implements tokens for Apigee Edge entities.
+ */
+
+use Drupal\apigee_edge\Entity\EdgeEntityInterface;
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Render\BubbleableMetadata;
+
+/**
+ * Implements hook_token_info_alter().
+ */
+function apigee_edge_actions_token_info_alter(&$info) {
```

Comment:
Yes we need implement custom token hooks because Edge entities are not Content entities. 

I'll look into how we can cover the available variables via documentation.